internal/handler/wallet/get.go

```diff
@@ -14,26 +14,25 @@ import (
 
 // NewGet instantiates a Get Handler
 // The "classical" way is not possible because we need to pass to the handler a wallet.WalletPrompterInterface.
-func NewGet(prompterApp prompt.WalletPrompterInterface) operations.RestWalletGetHandler {
+func NewGet(prompterApp prompt.WalletPrompterInterface) operations.RestAccountGetHandler {
 	return &walletGet{prompterApp: prompterApp}
 }
 
 type walletGet struct {
 	prompterApp prompt.WalletPrompterInterface
 }
 
-func (g *walletGet) Handle(params operations.RestWalletGetParams) middleware.Responder {
+func (g *walletGet) Handle(params operations.RestAccountGetParams) middleware.Responder {
 	wlt, err := wallet.Load(params.Nickname)
```

Comment:
No check for empty ? Maybe it's in the swagger spec ?

Reply:
yes if you call /rest/accounts/ it will fall in the List account handler

Reply:
you should then add a small comment